GameServerLib/Logic/Players/PlayerManager.cs

```diff
@@ -13,6 +13,8 @@ public class PlayerManager
 
         private List<Pair<uint, ClientInfo>> _players = new List<Pair<uint, ClientInfo>>();
         private int _currentId = 1;
+        private uint _blueUserId = 0;
```

Comment:
Don't do this. Set up a `Dictionary<TeamId, int>` and use that as the counter for all teams